### DISCUSSION TITLE
Cara mengakses docker dari vps?

### DISCUSSION THREAD

**Discussion by za4u** [deaafrizal/forum#39] [Troubleshooting Room]
Selamat malam bang, saya lagi belajar buat crud sederhana didocker dan saya ingin menggunakannya diserver vps. saya buat menggunakan codeigniter dan mongodb. 
pertama saya buat container mongo dan saya akses didocker desktop ( PC localhost) ketika diakses di mongodb compass dengan localhost bisa 
trus saya coba di vps (server) diakses menggunakan alamat ip ternyata tidak bisa
dari masalah saya,dikatakan saat buat/jalankan container harus memanggil port supaya bisa diakses dari luar dengan alamat ip.
kode yang saya gunakan untuk docker:
`docker container create --name mongoserver -p 27017:27017 -d mongo`
`docker container start mongoserver`
tapi tetap tidak bisa diakses diMongo Compass dengan ip. contoh: 147.139.222.xxx:27017
apakah command saya salah? dan apa command yang seharusnya, mohon bantuannya 
terima kasih

**Comment by deaafrizal** [comment]
seharusnya sih udah oke kok. cuma mungkin di cek dlu "docker ps" apa beneran udah nyala atau belum. terus pastiin ip, hostname, port itu sesuai juga pas mau testingnya
